src/ui/pages/NotificationDetails/components/ReceiveCredential/ReceiveCredential.tsx

```diff
@@ -28,7 +28,8 @@ const ReceiveCredential = ({
   activeStatus,
   notificationDetails,
   handleBack,
-}: NotificationDetailsProps) => {
+  multisigExn,
+}: NotificationDetailsProps & { multisigExn?: boolean }) => {
```

Comment:
Can you update `NotificationDetailsProps` rather than joining them here. Since it will apply to more than just receiving credentials soon